pyvelociraptor/wrappers.py

```diff
@@ -0,0 +1,135 @@
+from pyvelociraptor.velo_pandas import DataFrameQuery
+from pyvelociraptor import LoadConfigFile
+import pandas as pd
+import time
+
+pd.set_option('display.max_colwidth', None)
+pd.set_option('display.max_columns', None)
+pd.set_option('display.max_rows', None)
+
+def run_artifact(hostname, artifact_name, config=None, artifact_parameters=None, limit=None, artifact_collect_name=None, verbose=True, timeout=600):
+    """
+    # [Instantiates artifact for specific hostname and returns it's results as a pandas DataFrame]
+    # Examples:
+
+    # >>> df = wrappers.run_artifact("MY_HOSTNAME", "Windows.System.Pslist")
+
+    # [+] Client ID found: C.869eb611eaa5b899
+    # [!] Running VQL:
+
+    #         SELECT collect_client(
+    #             client_id="C.869eb611eaa5b899", 
+    #             artifacts="Windows.System.Pslist") 
+    #         AS Flow FROM scope() 
+
+    # [+] Got artifact flow ID: F.BVSSIUHNPUV7E
+    # [!] Collecting data using following VQL:
+
+    #         SELECT * FROM source(
+    #             artifact='Windows.System.Pslist',
+    #             client_id="C.869eb611eaa5b899", flow_id='F.BVSSIUHNPUV7E') 
+
+    # [!] Artifact is running... 0s
+    # [!] Artifact is running... 5.3s
+    # [+] Done! 11.0s
+
+    # >>> wrappers.run_artifact("NON_EXISTENT_HOSTNAME", "Windows.System.Pslist")
+    # [-] Cannot find any Client ID by provided hostname.
+
+    # >>> wrappers.run_artifact("MY_HOSTNAME", "Windows.System.NonExist")
+    # [+] Client ID found: C.869eb611eaa5b899
+    # [!] Running VQL:
+
+    #         SELECT collect_client(
+    #             client_id="C.869eb611eaa5b899", 
+    #             artifacts="Windows.System.Pslist123") 
+    #         AS Flow FROM scope() 
+
+    # [-] Artifact not instantiated... Check name and parameters!
+
+    # Returns:
+    #     [pandas.DataFrane]: [Results of artifact execution.]
+    """    
+    if config is None:
+        config = LoadConfigFile()
+
+    # FINDING CLIENT_ID FROM HOSTNAME
+    cid_query = DataFrameQuery(f"""
+    SELECT client_id FROM clients(search="{hostname}")
```

Comment:
Also here
```sql
 DataFrameQuery(f"""
    SELECT client_id FROM clients(search=Hostname)
""",config=config, Hostname=hostname)
```

Reply:
Ah, wasn't aware of this parameter handling syntax. Will use it in future. Queries fixed.